core/modules/qdisp/ExecStatus.h

```diff
@@ -57,7 +57,8 @@ class ExecStatus {
                  PROVISION_OK, // ???
                  REQUEST, REQUEST_ERROR,
                  RESPONSE_READY, RESPONSE_ERROR,
-                 RESPONSE_DATA, RESPONSE_DATA_ERROR, RESPONSE_DATA_NACK,
+                 RESPONSE_DATA, RESPONSE_DATA_ERROR, RESPONSE_DATA_ERROR_OK,
+				 RESPONSE_DATA_ERROR_CORRUPT, RESPONSE_DATA_NACK,
```

Comment:
How about putting RESPONSE_DATA_NACK on the same line where it used to be, and putting RESPONSE_DATA_ERROR_CORRUPT and RESPONSE_DATA_ERROR_OK  on one line together? And hopefully you can line things up as before.